squashfs/index.html

```diff
@@ -710,7 +775,7 @@ <h5 id="inode-directory-extended">Extended Directory</h5>
                                     <tr>
                                         <td>index_count</td>
                                         <td>u16</td>
-                                        <td>The number of directory indexes. TODO: More Info. May be related to exports?</td>
+                                        <td>Number of directory index entries following the inode structure</td>
```

Comment:
So this is actually one less than the number of index entries which follow, correct?